packages/quantic/force-app/main/default/lwc/quanticSmartSnippet/quanticSmartSnippet.js

```diff
@@ -110,6 +126,11 @@ export default class QuanticSmartSnippet extends LightningElement {
     }
   }
 
+  updateSearchStatusState() {
+    this.feedbackSubmitted = false;
+    this.shouldBeInNeutralState = true;
+  }
```

Comment:
```suggestion
  updateSearchStatusState() {
    this.feedbackSubmitted = false;
    this.liked = false;
    this.disliked = false;
  }
```

Reply:
Why isn't this handled at the headless level instead?
Shouldn't the state of the SmartSnippet like/dislike reset on a new query instead? 

IMO we shouldn't have to handle on the Quantic side no? Otherwise we'll have the same issue within Atomic?

Shouldn't there simply be a line somewhere in here:
https://github.dev/coveo/ui-kit/blob/2851cb2a9413badf6999a420516a79b262e61c89/packages/headless/src/features/question-answering/question-answering-slice.ts#L84

that simply sets liked/disliked to false?

Reply:
Yes the state of the Smart Snippet like/dislike should be reset on query, and that's already the case, see [L103](https://github.com/coveo/ui-kit/blob/2851cb2a9413badf6999a420516a79b262e61c89/packages/headless/src/features/question-answering/question-answering-slice.ts#L103-L104)

So at the Headless level everything is setup properly,  the only case where we do not reset in Headless is when doing a search query that returns the same answer, which I think make sense and is like that by design.

So that's why this issue is just a Quantic issue, we do not have it in Atomic mostly because the component re-rendering logic differs slightly. 

I updated the solution in Quantic to make it better  much better.
Let me know what you think.

Reply:
🆗